### PR TITLE
Added react-big-schedule.yml

### PR DESCRIPTION
**Pull request by ansulagrawal**
Hello! I think my project would be a great fit for Up the Grabs.

Library link: https://github.com/react-scheduler/react-big-schedule

https://github.com/up-for-grabs/up-for-grabs.net/issues/4852

### CODE REVIEW

**Comment by ansulagrawal** [comment]
@nemequ or anyone does this pr have any issues in it?

File: _data/projects/react-big-schedule.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- resource-scheduler
+- day-view
+- week-view
+- month-view
+- customizable
+- theming
+- responsive-design
+upforgrabs:
+  name: good first issue
+  link: https://github.com/react-scheduler/react-big-schedule
```
Comment (shiftkey) [inline _data/projects/react-big-schedule.yml]:
```suggestion
  link: https://github.com/react-scheduler/react-big-schedule/labels/good%20first%20issue
```

File: _data/projects/react-big-schedule.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+desc: React Big Schedule is a powerful and intuitive scheduler and resource planning solution built with React. Seamlessly integrate this modern, browser-compatible component into your applications to effectively manage time, appointments, and resources. With drag-and-drop functionality, interactive UI, and granular views, react-big-schedule empowers users to effortlessly schedule and allocate resources with precision. Enhance productivity and streamline your workflow with this React-based solution, designed to optimize time management and simplify calendar-based operations. Perfect for applications requiring advanced scheduling capabilities, react-big-schedule offers a seamless and intuitive experience for managing appointments, resource allocation, and time slots.
+site: https://react-big-schedule.vercel.app
+tags:
+- scheduling
+- calendar
+- react-component
+- javascript-library
+- open-source
+- frontend
+- UI-component
```
Comment (shiftkey) [inline _data/projects/react-big-schedule.yml]:
```suggestion
- ui-component
```

**Comment by shiftkey** [comment]
Leaving this open for PR author and others to adjust the config if they have a different label to use...

**Comment by ansulagrawal** [comment]
Atleast add hacktoberfest-accepted in the pr please.

**Review by shiftkey — APPROVED** [review]

**Pull request merged**